app/.babelrc

```
@@ -0,0 +1,4 @@
+{
```

Pulling in from https://github.com/navapbc/frontend/blob/main/examples/nextjs-uswds-and-cmsds/.babelrc
```suggestion
{
  /**
   * `next/babel` is required for our React tests to work
   * https://nextjs.org/docs/advanced-features/customizing-babel-config
   */
```